### PR TITLE
Fix Collaborations

### PR DESCRIPTION
**Pull request by sha14r**
## Description

Fixes: [Ticket](https://www.notion.so/hicommonwealth/Collaborations-doesn-t-show-new-collaborators-or-let-them-fix-edit-thread-a7728238d48c45919d42703cc258d17b)

## How has this been tested?

Collaborator sees edit option: 
image

Entry in the Collaboration table: 
image

## Have proper tags been added (for bug, enhancement, breaking change)?
- [ ] yes

## Does this PR affect any server routes?
- [ ] yes, and they are tested: [enter the % coverage here]
- [ ] yes, and they are not tested: [enter the % coverage here]
- [ ] yes, but I did not run tests
- [ ] no

### CODE REVIEW

**Comment by zakhap** [comment]
This PR is good for the DB! but in the client we need to make sure the data is being stored properly for the tooltip to work. See below what happens when I add a collaborator and don't refresh (it works on refresh):

Screen Shot 2022-03-17 at 12 38 29 PM
Screen Shot 2022-03-17 at 12 38 42 PM

**Comment by sha14r** [comment]
> This PR is good for the DB! but in the client we need to make sure the data is being stored properly for the tooltip to work. See below what happens when I add a collaborator and don't refresh (it works on refresh):
> 
> Screen Shot 2022-03-17 at 12 38 29 PM
> 
> Screen Shot 2022-03-17 at 12 38 42 PM

https://docs.google.com/document/d/1YpLSM3wX2k0X5zCw3WqhqSta84-JKR9vP0BzcVfNtXM/edit

File: server/routes/addEditors.ts [hunk 1]
```diff
@@ -106,47 +117,57 @@ const addEditors = async (models: DB, req: Request, res: Response, next: NextFun
...
+          },
+          req.wss,
+          [author.address]
+        );
+      })
     );
-  }));
 
-  const finalEditors = await models.Collaboration.findAll({
+  const finalCollaborations = await models.Collaboration.findAll({
```
Comment (gdjohnson) [inline server/routes/addEditors.ts:153]: Linters making it tough to see which changes are real—main backend fixes are in lines 153-170 of this file (/addEditors), and 68-84 of /deleteEditors route.

**Review by zakhap — APPROVED** [review]
Glad this finally got working! Thanks @gdjohnson for carrying this over the finish line.

**Pull request merged**